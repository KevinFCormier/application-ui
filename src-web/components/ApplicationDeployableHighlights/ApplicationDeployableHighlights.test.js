/*******************************************************************************
 * Licensed Materials - Property of IBM
 * (c) Copyright IBM Corporation 2016, 2019. All Rights Reserved.
 *
 * Note to U.S. Government Users Restricted Rights:
 * Use, duplication or disclosure restricted by GSA ADP Schedule
 * Contract with IBM Corp.
 *******************************************************************************/

const React = require('../../../node_modules/react')
const renderer = require('../../../node_modules/react-test-renderer')
const ApplicationDeployableHighlights = require('../ApplicationDeployableHighlights')
  .default

describe('ApplicationDeployableHighlights', () => {
  it('ApplicationDeployableHighlights renders correctly.', () => {
    const tree = renderer.create(<ApplicationDeployableHighlights />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
