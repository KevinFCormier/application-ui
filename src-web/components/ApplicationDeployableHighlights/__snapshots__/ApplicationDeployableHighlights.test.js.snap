// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ApplicationDeployableHighlights ApplicationDeployableHighlights renders correctly. 1`] = `
<div
  id="ApplicationDeployableHighlights"
>
  <div
    className="deployable-highlights-header"
  >
    Deployable Highlights
  </div>
  <div
    className="deployable-highlights-container"
  >
    <div
      className="deployable-highlights-info"
    >
      <div
        id="CountsCardModule"
        style={Object {}}
      >
        <div
          className="card-container"
        >
          <div
            className="card-container-content"
          >
            <div
              className="card-count-type"
              onClick={[Function]}
              onKeyPress={[Function]}
              role="button"
              tabIndex="0"
            >
              <div
                className="card-count"
              >
                5
              </div>
              <div
                className="card-type"
              >
                <div>
                  Versions
                </div>
              </div>
            </div>
            <div
              className="card-count-type"
              onClick={[Function]}
              onKeyPress={[Function]}
              role="button"
              tabIndex="0"
            >
              <div
                className="card-count"
              >
                3
              </div>
              <div
                className="card-type"
              >
                <div>
                  Completed
                </div>
              </div>
            </div>
            <div
              className="card-count-type"
              onClick={[Function]}
              onKeyPress={[Function]}
              role="button"
              tabIndex="0"
            >
              <div
                className="card-count"
              >
                1
              </div>
              <div
                className="card-type"
              >
                <div>
                  Failed
                </div>
              </div>
            </div>
            <div
              className="card-count-type"
              onClick={[Function]}
              onKeyPress={[Function]}
              role="button"
              tabIndex="0"
            >
              <div
                className="card-count"
              >
                1
              </div>
              <div
                className="card-type"
              >
                <div>
                  In Progress
                </div>
              </div>
            </div>
            <div
              className="card-count-type"
              onClick={[Function]}
              onKeyPress={[Function]}
              role="button"
              tabIndex="0"
            >
              <div
                className="card-count"
              >
                0
              </div>
              <div
                className="card-type"
              >
                <div>
                  Pending
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      className="deployable-highlights-incidents"
    >
      <div
        id="CountsCardModule"
        style={Object {}}
      >
        <div
          className="card-container"
        >
          <div
            className="card-container-content"
          >
            <div
              className="card-count-type"
              onClick={[Function]}
              onKeyPress={[Function]}
              role="button"
              tabIndex="0"
            >
              <div
                className="card-count"
              >
                1
              </div>
              <div
                className="card-type"
              >
                <div>
                  Incidents
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
